src/Serilog.Sinks.Exceptionless/Sinks/Exceptionless/ExceptionlessSink.cs

```diff
@@ -89,6 +96,10 @@ public class ExceptionlessSink : ILogEventSink, IDisposable {
 
             var builder = _client.CreateFromLogEvent(logEvent);
 
+            if (_defaultTags != null && _defaultTags.Any()) {
+                builder.AddTags(_defaultTags);
+            }
```

Comment:
We could simplify this and remove the linq import by just calling AddTags as this check already happens: https://github.com/exceptionless/Exceptionless.Net/blob/8306e83266fae60087bc0f022a7bec3e0eea013a/src/Exceptionless/Extensions/EventExtensions.cs#L211

Reply:
Duplicated check removed.